db/schema.rb

```diff
@@ -10,8 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_04_30_120750) do
-
+ActiveRecord::Schema[7.0].define(version: 2022_04_30_120750) do
```

Comment:
Hey @Sarthaknegigit, please revert changes in this file.

Reply:
I am on it.

Reply:
@pulkit-30  Resolved the changes in the last commit, please check.

Reply:
still working on the changes